Gemfile.lock

```diff
@@ -377,6 +377,3 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
   unicorn
   web-console (~> 2.0)
-
-BUNDLED WITH
-   1.10.6
```

Comment:
Please keep this change! Bundle has started to track the version number.
bundler/bundler#3697